### PR TITLE
Fix migrations when run against a non default database.

### PR DESCRIPTION
**Pull request by niall-mediasuite**
We are using this library on a project that is running tests with a separate test database. Previously due to the missing
`using` statement it would instead try to apply the `populate_prefix_hashed_key` using the default database connection.

### CODE REVIEW

**Comment by florimondmanca** [comment]
@niall-mediasuite Thanks for submitting this. Do you think of any changes we could make to our test setup to reflect this fix?

**Comment by niall-mediasuite** [comment]
Running the following command should allow you to test the changes. python `test_project/manage.py migrate --database test` I haven't added it to any of the scripts because I'm not sure where it would make sense. After the pytest? I have also updated the migrations for the test app as they also had the same issue.

**Comment by niall-mediasuite** [comment]
Please let me know if there is anything more I can do to help get this merged.

**Comment by florimondmanca** [comment]
* Can you confirm the 1.4 Migration page https://florimondmanca.github.io/djangorestframework-api-key/upgrade/1.4/ would get updated too?
* I'm just trying to mentally go through whether this requires any action from folks with custom API key models once released. Should people go and modify their in-repo copies of the migration too? Will anything break if they keep the old version? Will a run of `makemigrations` generate a new migration due to this change? Etc. If you can run through your line of thought about this, that'd be convincing, and then I guess we can merge and release. :+1:

**Comment by niall-mediasuite** [comment]
Yeah the code in that migration page should also be updated.

My understanding is that if someone has previously created these migrations then they won't have any issues unless they then switch to store the API keys in the non default database. So it would probably be wise to updated the docs and add a section about the change that's required. 

`makemigrations` won't be impacted at all by this change as its used to generate new migrations from model changes, not changes to migrations themselves.

File: test_project/project/settings.py [hunk 1]
```diff
@@ -55,7 +55,10 @@
 
 # Database
 
-DATABASES = {"default": dj_database_url.config(default="sqlite://db.sqlite3")}
+DATABASES = {
+    "default": dj_database_url.config(default="sqlite://:memory:"),
+    "test": dj_database_url.config(default="sqlite://:memory:"),
+}
```
Comment (florimondmanca) [inline test_project/project/settings.py:61]: I don't think we want to use in-memory instances here — the `test_project` is meant to be played with locally, with persisting data.

So I think we want this instead…?

```suggestion
DATABASES = {
    "default": dj_database_url.config(default="sqlite:///db.sqlite3"),
    "test": dj_database_url.config("TEST_DATABASE_URL", default="sqlite://test.sqlite3"),
}
```

**Review by florimondmanca — COMMENTED** [review]
Thanks! I ran this against a `test_project` modified as per my comment here, and I could verify that it works, whereas on `master` I'd get an `AttributeError`.

**Comment by florimondmanca** [comment]
Closing in favor of #215.

**Pull request closed without merging**